src/test/java/uk/gov/hmcts/reform/pip/data/management/service/ExcelConversionServiceTest.java

```diff
@@ -0,0 +1,171 @@
+package uk.gov.hmcts.reform.pip.data.management.service;
+
+import org.assertj.core.api.SoftAssertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.web.multipart.MultipartFile;
+import org.testcontainers.shaded.com.fasterxml.jackson.core.type.TypeReference;
+import org.testcontainers.shaded.com.fasterxml.jackson.databind.ObjectMapper;
+import org.testcontainers.shaded.org.apache.commons.io.IOUtils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Map;
+
+@ActiveProfiles("test")
+class ExcelConversionServiceTest {
```

Comment:
If a header row does not exist, will it just choose the first row as the header?

Just trying to work out if we need a specific test for this scenario

Reply:
If header row does not exist, it will take the first row as header and later on will fail the validation because the header row does not match that for the schema